src/app/query/bitcoin/ordinals/ordinals-aware-utxo.query.ts

```diff
@@ -73,7 +73,7 @@ export function useOrdinalsAwareUtxoQueries(utxos: TaprootUtxo[] | btc.Transacti
         queryKey: makeOrdinalsAwareUtxoQueryKey(txId, txIndex),
         queryFn: () => fetchOrdinalsAwareUtxo(txId, txIndex),
         select: (resp: OrdApiInscriptionTxOutput) =>
```

Comment:
Is this OK? Looks like it could be a prettier error moving the bracket?

Reply:
Prettier 3.0 🆕